### ISSUE TITLE
CMake warning: Installing polars in a venv failed

### ISSUE THREAD

**Issue by MakisH** [precice/precice#1766]
**Describe your setup**

Operating system (e.g. Linux distribution and version): _Ubuntu 22.04_
preCICE Version: develop on 1678f9fb8a441c959723fea36b440706bd127b88

Add the versions/commit hashes of further adapter and/or bindings.

**Describe the problem**

When configuring preCICE, I get this new warning:

```
Installing polars in a python venv to allow for full profiling tests.
CMake Warning at tools/profiling/CMakeLists.txt:33 (message):
  Installing polars in a venv failed.
```

Probably we can improve or document something here.

**Step To Reproduce**

```
mkdir build
cd build
cmake ..
```

**Comment by IshaanDesai** [comment]
On a Ubuntu 22.04 machine with state 8393841e62a78e87db1067543e2da4cc265e55d5 I do not see the warning, but only:

```bash
Installing polars in a python venv to allow for full profiling tests.
```

**Comment by fsimonis** [comment]
@IshaanDesai this should fix part of the issue https://github.com/precice/precice/pull/1773

**Comment by MakisH** [comment]
> @MakisH The error output of the venv installation is enabled. So it succeeds, but the detection fails. Very weird indeed. I enabled the error output of the detection itself in #1773. Does that help in some way?

No, it does not help. Answered in the PR: https://github.com/precice/precice/pull/1773#issuecomment-1677449711

**Comment by fsimonis** [comment]
Closed via #1773

**Comment by davidscn** [comment]
The current behavior of preCICE is still somewhat unsatisfactory. Without asking the user or even giving the user an option, preCICE tries to install polars in a virtual environment. However, if the current machine has no internet access (SuperMUC), the installation hangs

```bash
Installing polars in a python venv to allow for full profiling tests.
WARNING: Retrying (Retry(total=4, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<pip._vendor.urllib3.connection.VerifiedHTTPSConnection object at 0x7f5c9f621eb8>: Failed to establish a new connection: [Errno 101] Network is unreachable',)': /simple/pip/
...
```

One could in principle disable `PRECICE_BUILD_TOOLS` to bypass the installation, but then I don't get any tools. Polars is only required for a small subset of the tools. HPC users would most certainly still want to use the merging capabilities of `precice-events`, for example.

**Comment by fsimonis** [comment]
Closed via #1808. We now detect polars instead of installing it.

I changed the CI to install polars via pip in ubuntu images.
